Dockerfile

```diff
@@ -15,6 +15,7 @@ COPY . /opt/reopt
 # Install python packages
 WORKDIR /opt/reopt
 RUN ["pip", "install", "-r", "requirements.txt"]
+RUN ["pip", "install", "django-rename-app@git+https://github.com/tonioo/django-rename-app@10c87d62bfc14853379f6a0505f53ba4532848ab"]
```

Comment:
@Bill-Becker will this line need to be reverted once merged into master?

Reply:
Yes, good point